### PR TITLE
Updated Algorithm Check for Palindromes

### PR DESCRIPTION
**Pull request by atjonathan**
Refer #1094

### CODE REVIEW

File: Algorithm-Check-For-Palindromes.md [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
 
 - If you are unsure of what a palindrome is, it is a word or phrase that when reversed spells the same thing forwards or backwards. A simple example is `mom`, when you reverse the letters, it spells the same thing! Another example of a palindrome is `race car`. When we take out anything that is not a character it becomes `racecar` which is the same spelled forwards or backwards!
 
 Once we have determined whether it is a palindrome or not we want to return either `true` or `false` based on our findings.
 
-## Hint: 1
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Check-For-Palindromes.md]: Remember to add links....

File: Algorithm-Check-For-Palindromes.md [hunk 2]
```diff
@@ -47,7 +59,23 @@ function palindrome(str) {
...
+- Next we _chain_ `.toLowerCase()` to remove any capital letters because `A` is a different character than `a`. The problem did not ask us to worry about making sure the case of the characters was identical, just the spelling.
+
+- Our next step is to take our string and `.split()` it, `.reverse()` it, and finally `.join()` it back together.
+
+- Last step is to check that the string is the same forwards and backwards and return our result!
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Check-For-Palindromes.md]: Remember to add links....

File: Algorithm-Check-For-Palindromes.md [hunk 3]
```diff
@@ -63,9 +91,21 @@ function palindrome(str) {
...
+- We start by using the same methods of replacing characters we don't want in the string using `RegEx`'s, regular expressions, and then make our string lowercase.
+
+- Next we set up our `for` loop and declare an index `i` to keep track of the loop. We set our escape sequence to when `i` is greater than the length of the string divided by two, which tells the loop to stop after the halfway point of the string. And finally we set `i` to increment after every loop.
+
+- Inside of each loop we want to check that the letter in element `[i]` is equal to the letter in the length of the string minus i, `[str.length - i]`. Each loop, the element that is checked on both sides of the string moves closer to the center until we have checked all of the letters. If at any point the letters do not match, we return `false`. If the loop completes successfully, it means we have a palindrome and therefore we return `true`!
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Check-For-Palindromes.md]: Remember to add links....

File: Algorithm-Check-For-Palindromes.md [hunk 4]
```diff
@@ -1,42 +1,54 @@
 # Algorithm Check for Palindromes
 
-![](https://i.imgur.com/ozDWKEi.jpg)
+![](https://i.imgur.com/ozDWKEi.jpg)  
 
-### Explanation:
+### :checkered_flag: Problem Explanation:
```
Comment (Rafase282) [inline Algorithm-Check-For-Palindromes.md]:
```diff
-### :checkered_flag: Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
+
+### :checkered_flag: Problem Explanation:
+
```

**Comment by Rafase282** [comment]
# General links
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-Replace
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-ToLowerCase

# solution 1
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-Split
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Reverse
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Join

# Solution 2
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Regex-Resources

# Solution 3
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-Slice

Please remove the "relevant links" from the hints section. and add the `> _try to solve the problem now_`

**Comment by Rafase282** [comment]
Better yet, merge this https://github.com/atjonathan/wiki/pull/2

**Comment by Rafase282** [comment]
# LGTM :+1:

**Pull request merged**